INSTALL

```diff
@@ -577,6 +583,12 @@ The specific libraries and their options are:
   Options: SUPERLU_OPT, SUPERLU_LIB.
   Versions: SuperLU_DIST >= 5.1.0.
 
+- MUMPS (optional), used when MFEM_USE_MUMPS = YES. Note that MUMPS
+  requires LAPACK, SCALAPACK and a reordering package such as PORD or METIS.
+  URL: http://mumps.enseeiht.fr
+  Options: MUMPS_OPT, MUMPS_LIB.
+  Versions: MUMPS >= 5.1.1
```

Comment:
`CHANGELOG` says `versions >= 5.2.0` -- which one is correct?

Reply:
it is 5.1.1. I fixed it.